# Ballouz lab publications

| Nickname(s)                                                                      |    Full citation  and link                                                                                                                                                                                                                                                                                                                                                 |    Main   Takeaways/Comments                                                                                                                                                                                                                                                                                                                        |    Keywords                                                                                           |
|----------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------|
|  Single-cell transcriptomics                                                     |    Ballouz      | 



## Key pre-Ballouz lab papers (in chronological order): 

| Nickname(s)                                                                      |    Full citation and link                                                                                                                                                                                                                                                                                                                               |    Main   Takeaways/Comments                                                                                                                                                                                                                                                                                                                                                     |    Keywords                             |
|----------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------|
|    The   Guidance Paper                                                          |    Ballouz   S, Verleyen W, Gillis J. Guidance for RNA-seq co-expression network   construction and analysis: safety in numbers. Bioinformatics. 2015. doi: 10.1093/bioinformatics/btv118.   PubMed PMID: 25717192.   https://academic.oup.com/bioinformatics/article/31/13/2123/196230                                                                                  |    -            Itâ€™s important to have lots of data   -            Microarray coexpression and RNA-seq coexpression are   similar except that low expressing genes form strong modules in microarray   but not RNA-seq networks                                                                                                                     |    RNA-seq,   microarray, coexpression, human, replicability, network analysis                        |
|    AuPairWise                                                                    |    Ballouz   S, Gillis J. AuPairWise: A Method to Estimate RNA-Seq Replicability through   Co-expression. PLoS computational biology. 2016;12(4):e1004868. doi:   10.1371/journal.pcbi.1004868. PubMed PMID: 27082953; PubMed Central PMCID:   PMC4833304.   http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1004868                               |    -         Higher coexpression of selected gene-pairs over random   gene-pairs can be used for RNA-seq quality control                                                                                                                                                                                                                            |    Software,   coexpression                                                                           |
|    EGAD                                                                          |    Ballouz   S, Weber M, Pavlidis P, Gillis J. EGAD: ultra-fast functional analysis of   gene networks. Bioinformatics. 2017 Feb 15; 33(4):612-614. PubMed PMID:   27993773.   https://academic.oup.com/bioinformatics/article/33/4/612/2664343                                                                                                                          |    -         Bioconductor package for neighbor voting and other   assorted functions                                                                                                                                                                                                                                                                |    Software,   network analysis                                                                       |
|    ErmineJ                                                                       |    Ballouz   S, Pavlidis P, Gillis J. Using predictive specificity to determine when gene   set analysis is biologically meaningful. Nucleic Acids Research. 2016. doi:   10.1093/nar/gkw957    https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5389513/                                                                                                                    |    -         Specificity and robustness are useful heuristics to   identify reliable enrichment results.     -         We can use multifunctionality as a way of targeting   specificity and robustness.                                                                                                                                            |    Enrichment   analysis, GO                                                                          |
|    The   Single Cell Coexpression Paper (The Genome Biology Paper)               |    Crow   M, Paul A, Ballouz S, Huang ZJ, Gillis J (2016) Exploiting single-cell   expression to characterize co-expression replicability. Genome Biology 17,   101. PubMed PMID: 27165153; PubMed Central PMCID: PMC4862082.   https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0964-6                                                               |    -            Single cell RNA-seq coexpression aggregation ~ bulk   -            Coexpression within cell types ~ across cell types   -            Expression level can predict coexpression, so should test   for this                                                                                                                           |    Single   cell, meta-analysis, coexpression, Brainspan, control experiments, novel data             |
|    The   Effect Size Paper (The Genome Medicine Paper)                           |    Ballouz   S, Gillis J. Strength of functional signature correlates with effect size in   autism. Genome Med. 2017 Jul 7; 9(1):64. PubMed PMID: 28687074; PubMed   Central PMCID: PMC5501949.   https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-017-0455-8                                                                                            |    -         The more strongly a gene is associated with a disease,   the more likely it is to show functional convergence.                                                                                                                                                                                                                         |    Expression,   functional enrichment, disease, genetics, autism, Brainspan                          |
|    MetaNeighbor                                                                  |    Crow   M, Paul A, Ballouz S, Huang ZJ, Gillis J. Characterizing the replicability of   cell types defined by single cell RNA-sequencing data using MetaNeighbor.   Nature communications. 2018; 9(1):884. PMID: 29491377, PMCID: PMC5830442   https://www.nature.com/articles/s41467-018-03282-0                                                                      |    -            Cell type transcriptional profiles are replicable across   studies   -            When predicting cell identity, almost any set of genes   can be used to improve performance above chance   -            Highly variable genes are generally useful, even when   cell types are rare or only subtly different from the outgroup    |    Single   cell, meta-analysis, brain, software                                                      |
|    Aligner                                                                       |    Ballouz   S, Dobin A, Gingeras TR, Gillis J. The fractured landscape of RNA-seq   alignment: The default in our STARs. Nucleic Acids Research.     https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gky325/4990636                                                                                                                                        |    -            Exact expression is hard to get right, statistical   differences are easy   -            Most parameter choices are fine, but our ways of telling   what is fine are overly technical.                                                                                                                                              |    RNA-seq,   STAR, software, meta-analysis, collaboration                                            |
|    Consensus (null) opinion                                                      |    Ballouz S, Dobin A, Gillis J. (2019) Is it time to change the reference genome? Genome Biology.     https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1774-4                                                                                                                                      |                                                                                                                                         - The reference genome is idiosyncratic and shouldn't be used as a baseline.  - Incorporating the most frequent/common allele into the reference (i.e., converting it into a 'consensus' genome) is a good-enough fix     |   Consensus genome, Reference genome, mapping, variant-calling, collaboration
